### ISSUE TITLE
Global XSLT transform is always applied to reports in tests tab even when disabled in Ladybug options or a specific XSLT transform is defined on the test

### ISSUE THREAD

**Issue by MLenterman** [wearefrank/ladybug#210]
For the saved reports under the test tab, it currently seems to be impossible to not have the global XSLT transformation being applied to it. This restricts the test to always only compare the first and last checkpoint. The specific XSLT transformation that can be added on the test itself, is applied on the already transformed first and last checkpoint.

Having a default XSLT transformation that filters on first and last checkpoint is nice to have, but I would say that adding a specific transformation on the test itself should override the default transformation instead of being applied on top of the default transformation

**Comment by checkiecheck** [comment]
@MatthijsSmets can you have a look at this and see what options we can do?

**Comment by MLenterman** [comment]
> But you can open a report from the test tab to its own tab and add a custom transformation there right?
> ### First open the report in a new tab with the button:
> 
> ![Image](https://private-user-images.githubusercontent.com/93487259/395619640-71a690d4-0828-4ce0-9a9c-374ed6d3653b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0NDk5MTQsIm5iZiI6MTczNDQ0OTYxNCwicGF0aCI6Ii85MzQ4NzI1OS8zOTU2MTk2NDAtNzFhNjkwZDQtMDgyOC00Y2UwLTlhOWMtMzc0ZWQ2ZDM2NTNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDE1MzMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJkMWJlOWExOGVhODYwNGZlZDc1NzY4MDg1MGExMmU2MTc2MjkyYTY4MGI5NzM2ZWNjNzI0YjUwNmNlMzliMGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.uVMJ7Rz6Qun7pxOr7_PviH9-82kg_o-5Gn5ecQqqSa4)
> ### After that switch to the edit view, make sure the root node is selected:
> 
> ![Image](https://private-user-images.githubusercontent.com/93487259/395619904-f501642d-775c-484b-8bd9-a9332769b7f4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0NDk5MTQsIm5iZiI6MTczNDQ0OTYxNCwicGF0aCI6Ii85MzQ4NzI1OS8zOTU2MTk5MDQtZjUwMTY0MmQtNzc1Yy00ODRiLThiZDktYTkzMzI3NjliN2Y0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDE1MzMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE2YzY0MjU5MzIxNjViZDE0NTUzZTFlMGVmNzg3YWU4ZDZlMjU5NWY3OTQ5MDI1YjcyM2ZhZDdkYjFhMDNlYjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.kraczxmca8TrhWoL5V0yUqvmM-3yaGNM_byr69Dtah0)
> ### You'll see the transformation form field where you can enter a custom transformation for that report, don't forget to save😊.
> 
> ![Image](https://private-user-images.githubusercontent.com/93487259/395621319-c8908359-a94c-425f-995b-0efe0312dc96.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0NDk5MTQsIm5iZiI6MTczNDQ0OTYxNCwicGF0aCI6Ii85MzQ4NzI1OS8zOTU2MjEzMTktYzg5MDgzNTktYTk0Yy00MjVmLTk5NWItMGVmZTAzMTJkYzk2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE3VDE1MzMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlhMDM0NTc5MjljY2FhODk1MjA0YzlkODFmMGViZjBkOWRiZDJhNzU2Mjk3ZGQwYjA4NmVkM2Y5Y2U5OTU3MjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.UR4RKdRqazA7uyOp3fA_106GlbDCOuIFMU9l0WszGd8)

That is correct, but that transform is only applied on the first and last checkpoint. If you put a one-on-one copy as xslt transform on the testcase itself like you showed, you will see that only first and last checkpoint is there. It should be all checkpoints within the report.

**Comment by MatthijsSmets** [comment]
Ah I understand @jacodg can you have a look at this?
